library/core/src/main/java/com/google/android/exoplayer2/mediacodec/MediaCodecInfo.java

```diff
@@ -572,6 +581,49 @@ public boolean isAudioChannelCountSupportedV21(int channelCount) {
     return true;
   }
 
+  /**
+   * On versions M and below, VP9 codecCapabilities do not advertise profile level
+   * support. In this case, estimate the level from MediaCodecInfo.VideoCapabilities
+   * instead. For more information, consult
+   * https://developer.android.com/reference/android/media/MediaCodecInfo.CodecProfileLevel.html
+   */
+  private static CodecProfileLevel[] getVp9CodecProfileLevelsV23(CodecCapabilities capabilities) {
+    // https://www.webmproject.org/vp9/levels
+    final int[][] bitrateMapping = {
+        {180000, CodecProfileLevel.VP9Level52},
```

Comment:
I think the bitrate values here need to be 1000x larger, since the table at `https://www.webmproject.org/vp9/levels` is in `x1000 bit/s`.

Reply:
Ah, thanks for catching this! So this is not correct for chromium media either...

Reply:
It looks to me like both the bitrate constants _and_ the profile level constants are wrong in the Chromium code you reference. I could be wrong, but it just looks like it doesn't do the right thing, as far as I can see!